src/journey-maps/angular/services/constants.ts

```diff
@@ -1,6 +1,14 @@
+import { LngLatBoundsLike } from 'maplibre-gl';
+
 export const SBB_CLUSTER_RADIUS = 50;
 export const SBB_MARKER_BOUNDS_PADDING = 40;
 
+export const SBB_BOUNDING_BOX: LngLatBoundsLike = [
```

Comment:
Can you change it to

    [
        [5.7349, 45.6755],
        [10.6677, 47.9163],
    ]

I already did this in my [PR](https://github.com/sbb-design-systems/sbb-angular/pull/1479/files#diff-9842de4996dac2e5518672c8ac7e07b04610885540a1eb4f3e2245be9e5641ebR25). 

Because it actually should be [south-west][north-east].

Reply:
![image](https://user-images.githubusercontent.com/632091/171580386-353cea3e-da2a-4f4e-8727-ff0c3305dd34.png)

Reply:
https://tenor.com/bvL6X.gif

Reply:
<div class="tenor-gif-embed" data-postid="19927097" data-share-method="host" data-aspect-ratio="1" data-width="100%"><a href="https://tenor.com/view/mrbean-angry-sinirli-sinirli-olmak-gif-19927097">Mrbean Angry GIF</a>from <a href="https://tenor.com/search/mrbean-gifs">Mrbean GIFs</a></div> <script type="text/javascript" async src="https://tenor.com/embed.js"></script>

Reply:
![mrbean-angry](https://user-images.githubusercontent.com/632091/171580726-535a8b85-7717-4346-9b4a-42e4135f630b.gif)